### ISSUE TITLE
 On Windows: %APPDATA%/Claude/claude_desktop_config.json <-- does not exist

### ISSUE THREAD

**Issue by scott------** [Alec2435/python_mcp#1]
should we create this file?  I just installed fresh copy of windows claude desktop but that directory path and config file do not exist.

There is a config.json in C:\Users\mywindowsusername\AppData\Roaming\Claude but all that has in it is this:
{
	"darkMode": "dark",
	"scale": 0
}

**Comment by goodrahstar** [comment]
@scott------ While I'm not entirely sure about Windows, on Mac I had to manually create the config JSON file, and the directory was hidden by default. You might want to check if the folder is hidden on your system - you'll need to create the JSON file either way.

**Comment by scott------** [comment]
Yes you are right I had to create it.  Was a pain to setup on Windows but
it works :-) one thing I wish the anthropic readme said was you have to
turn on developer mode and you do that through help.  That would have saved
me like 10 mins.
